internal/resource/config/localidentity/local_identity_identity_profile_data_source.go

```diff
@@ -0,0 +1,671 @@
+								Required:    false,
+								Optional:    false,
+								Computed:    true,
+							},
+						},
+					},
+					"require_verified_email": schema.BoolAttribute{
+						Description: "Whether the user must verify their email address before they can complete a single sign-on transaction. The default is false.",
+						Required:    false,
+						Optional:    false,
+						Computed:    true,
+					},
+					"require_verified_email_template_name": schema.StringAttribute{
+						Description: "The template to render when the user must verify their email address before they can complete a single sign-on transaction. The default is local.identity.email.verification.required.html. Note:Only applicable if EmailVerificationType is OTL and requireVerifiedEmail is true.",
+						Required:    false,
+						Optional:    false,
+						Computed:    true,
+					},
+				},
+			},
+			"data_store_config": schema.SingleNestedAttribute{
+				Description: "The local identity profile data store configuration.",
+				Required:    false,
+				Optional:    false,
+				Computed:    true,
+				Attributes: map[string]schema.Attribute{
+					"base_dn": schema.StringAttribute{
+						Description: "The base DN to search from. If not specified, the search will start at the LDAP's root.",
+						Required:    false,
+						Optional:    false,
+						Computed:    true,
+					},
+					"type": schema.StringAttribute{
+						Description: "The data store config type.",
+						Required:    false,
+						Optional:    false,
+						Computed:    true,
+						Validators: []validator.String{
+							stringvalidator.OneOf([]string{"LDAP", "PING_ONE_LDAP_GATEWAY", "JDBC", "CUSTOM"}...),
+						},
+					},
+					"data_store_ref": schema.SingleNestedAttribute{
+						Description: "Reference to the associated data store.",
+						Required:    false,
+						Optional:    false,
+						Computed:    true,
+						Attributes: map[string]schema.Attribute{
+							"id": schema.StringAttribute{
+								Description: "The ID of the resource.",
+								Required:    false,
+								Optional:    false,
+								Computed:    true,
+							},
+							"location": schema.StringAttribute{
+								Description: "A read-only URL that references the resource. If the resource is not currently URL-accessible, this property will be null.",
+								Required:    false,
+								Optional:    false,
+								Computed:    true,
+							},
+						},
+					},
+					"data_store_mapping": schema.MapNestedAttribute{
+						Description: "The data store mapping.",
+						Required:    false,
+						Optional:    false,
+						Computed:    true,
+						NestedObject: schema.NestedAttributeObject{
+							Attributes: map[string]schema.Attribute{
+								"type": schema.StringAttribute{
+									Description: "The data store attribute type.",
+									Required:    false,
+									Optional:    false,
+									Computed:    true,
+									Validators: []validator.String{
```

Comment:
I don't think we need validators on any data source attributes that are computed